terraform/keycloak-sandbox/README.md

```diff
@@ -139,6 +139,15 @@ aws ssm get-parameters \
 Visit the Keycloak load balancer URL again, and sign into the admin UI using the
 username and password you just fetched.
 
+## Setup GovUK Notify
+
```

Comment:
Just wondering - is this step optional? Do you only need to do it if you want to generate emails from the Sandbox Keycloak? If so, it might be worth mentioning, so that it saves someone time if they're creating the instance to try out something else.

Reply:
Good point. Updated